<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clean Linux - ุชูุธูู ูุธุงู ููููุณ</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="toolbarWidget" native="true">
      <layout class="QHBoxLayout" name="toolbarLayout">
       <item>
        <widget class="QToolButton" name="analyzeButton">
         <property name="text">
          <string>๐ ุชุญููู ุงููุธุงู</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="cleanButton">
         <property name="text">
          <string>๐งน ุชูุธูู ุงูุขู</string>
         </property>
        </widget>
       </item>

       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="languageComboBox">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>English</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ุงูุนุฑุจูุฉ</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="darkModeButton">
         <property name="text">
          <string>๐</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="categoriesGroup">
      <property name="title">
       <string>ุงููููุงุช ุงูุชู ูููู ุชูุธูููุง</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="selectAllCheckbox">
         <property name="text">
          <string>ุชุญุฏูุฏ ุงููู</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="categoriesTable">
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="statsGroup">
      <property name="title">
       <string>ุงูุฅุญุตุงุฆูุงุช</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="totalSizeLabel">
         <property name="text">
          <string>ุงูุญุฌู ุงูุฅุฌูุงูู: 0 ุจุงูุช</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="selectedSizeLabel">
         <property name="text">
          <string>ุงูุญุฌู ุงููุญุฏุฏ: 0 ุจุงูุช</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="filesCountLabel">
         <property name="text">
          <string>ุนุฏุฏ ุงููููุงุช: 0</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="chartGroup">
      <property name="title">
       <string>ุงููุณุงุญุฉ ุงููุณุชุฑุฌุนุฉ</string>
      </property>
      <layout class="QVBoxLayout" name="chartLayout">
       <item>
        <widget class="QChartView" name="chartView"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>ููู</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>ุฅุนุฏุงุฏุงุช</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>ูุณุงุนุฏุฉ</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>ุฎุฑูุฌ</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>ุชูุถููุงุช</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>ุญูู</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
